Rakefile

```diff
@@ -0,0 +1,6 @@
+# Add your own tasks in files placed in lib/tasks ending in .rake,
```

Comment:
# Backend re-review

- [ ] OPTIONAL:

This section of the mobile view is still questionable:


![image](https://github.com/Kgomotso196/House-Booking-Frontend/assets/57421823/8d38ac0f-4504-4083-b4ec-81690f2e0e56)

The issue is that the content overflows the screen.

Kindly, I suggest you solve this issue before final evaluation. 👍